README.md

```diff
@@ -66,7 +67,7 @@ with your default Python environment and can easily use different versions of Py
 in different projects. You can create a virtual environment for ELFI using anaconda with:
 
 ```
-conda create -n elfi python=3.5 numpy
+conda create -n elfi python=3.5 matplotlib
```

Comment:
Hmm. doesnt't seem right. The linux anaconda distros seems to require numpy, so that's why its there. Can you move matplotlib installation to travis.yml?